examples/server/src/main.rs

```diff
@@ -1,10 +1,11 @@
 use async_std::io;
 use async_std::net::{TcpListener, TcpStream};
-use async_std::prelude::*;
+use async_std::stream::StreamExt;
 use async_std::task;
 use async_tls::TlsAcceptor;
-use rustls::internal::pemfile::{certs, rsa_private_keys};
-use rustls::{Certificate, NoClientAuth, PrivateKey, ServerConfig};
+use futures_lite::io::AsyncWriteExt;
```

Comment:
why do you need this over https://docs.rs/async-std/latest/async_std/io/trait.WriteExt.html ?

Reply:
`AsyncWriteExt` from the `futures_util` crate gives us `.close()` on the `TlsStream` which implements `AsyncWrite`. See: https://docs.rs/futures-util/latest/futures_util/io/trait.AsyncWriteExt.html#method.close

Sadly `WriteExt` from async_std doesn't provide `.close()`. See this related issue: https://github.com/async-rs/async-std/issues/977

Reply:
Sorry for getting back to you so late. Your review slipped through my notifications.

Reply:
I see, thanks for the info